util/prep-bazel-airgapped-build.sh

```diff
@@ -156,8 +156,14 @@ if [[ ${AIRGAPPED_DIR_CONTENTS} == "ALL" || \
     @rust_windows_x86_64_toolchains//...
   cp -R "$(${BAZELISK} info output_base)"/external/${BAZEL_PYTHON_WHEEL_REPO} \
     ${BAZEL_AIRGAPPED_DIR}/
-  cp -R "$(dirname "$(readlink -f "$(${BAZELISK} info output_base)"/external/${BAZEL_BITSTREAMS_REPO}/cache)")" \
-    ${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHEDIR}
+  # We don't need all bitstreams in the cache, we just need the latest one so
+  # that the cache is "initialized" and "offline" mode will work correctly.
+  mkdir -p ${BAZEL_AIRGAPPED_DIR}/${BAZEL_BITSTREAMS_CACHEDIR}/cache
+  cp ~/.cache/opentitan-bitstreams/latest.txt \
```

Comment:
nit: this path can be a readonly variable.

**NOTE:** bash doesn't expand `~` inside double-quotes, use "${HOME}....."

Reply:
fixed.